docs/cmd/tkn_pipeline_sign.md

```diff
@@ -0,0 +1,47 @@
+## tkn pipeline sign
+
+Sign Tekton Task/Pipeline
+
+### Usage
+
+```
+tkn pipeline sign
+```
+
+### Synopsis
+
+Sign Tekton Task/Pipeline
+
+### Examples
+
+Sign a Pipeline pipeline.yaml:
+	tkn pipeline sign pipeline.yaml -K=cosign.key -f=signed.yaml
+or using kms
+	tkn pipeline sign pipeline.yaml -K=gcpkms://projects/PROJECTID/locations/LOCATION/keyRings/KEYRING/cryptoKeys/KEY/cryptoKeyVersions/VERSION -f=signed.yaml
+
+### Options
+
+```
+      --allow-missing-template-keys   If true, ignore any errors in templates when a field or map key is missing in the template. Only applies to golang and jsonpath output formats. (default true)
+  -f, --file-name string              Skip verifying the payload'signature
```

Comment:
the description does not seem to match --file-name? 🤔

Reply:
😂  good catch